api/DTOs/FiltersDto.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+using System;
+namespace YonderfulApi.DTOs
+{
+  public class FiltersDto
+  {
+    [Required]
+    public DateTime StartingDate { get; set; }
+    
+    public DateTime? EndingDate { get; set; }
+    public String? Category { get; set; }
```

Comment:
Should change this to lower case string - see here why: https://stackoverflow.com/a/7077